documentation/kubernetes.md

```diff
@@ -7,6 +7,10 @@ In addition to its availability as a .NET CLI tool, the `dotnet monitor` tool is
 
 For Dockerfiles and repository information, see [Running in Docker](./docker.md)
 
+## Non-root considerations
+
+Starting with .NET 8.0, both the sample aspnet application and dotnet-monitor run as non-root. If both the application and dotnet-monitor are 8+, no additional configuration is required. Otherwise, a [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) configuration may need to be added to the application, dotnet-monitor, or both.
```

Comment:
@poppastring I can remove or shorten this section and move to any additional non-root docs we have